### DISCUSSION TITLE
Setting up nested Collections/Pages/Taxonomy?

### DISCUSSION THREAD

**Discussion by kcomplexes** [statamic/cms#8007] [Q&A]
Hi,

I'm having some difficulty understanding and setting up statamic for a personal travel website.
I have a landing page and blog going, but I would also like to have another section called 'Destinations'

1. The 'Destination' landing page would show links to all the countries I have visited.
2. Each 'Country' page would then list a region/city of that country I have visited while perhaps having some high level information about the country.
3. On each 'region' page would be the content (pictures, itinerary etc) of each region.

The tree view would be something like this:
- Destinations
  - Australia
    - Victoria
      - Melbourne
      - High Country
    - Adelaide 
  - Indonesia
    - Bali
      - Ubud 
    - Lombok
   - Canada 

I am trying to work out the most statamic and effective way of doing this. Would it be through Taxonomies? Collections? Mounts? Tags?

Thankyou!

**Comment by ryanmitchell** [comment]
I would just add it as another section within your 'pages' collection.
Is there any reason why that wouldn't work?
